go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/Azure/karpenter-provider-azure
 
 go 1.24.4
 
+require github.com/Azure/skewer v0.0.20
```

Comment:
Needs to be under require, just undo changes?

Reply:
updated